tests/integration_tests/consistent_replicate_gbk/run.sh

```diff
@@ -90,7 +90,7 @@ function run() {
 	run_sql "insert into consistent_replicate_gbk.GBKTABLE2 select * from consistent_replicate_gbk.GBKTABLE" ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 
 	# to ensure row changed events have been replicated to TiCDC
-	sleep 5
```

Comment:
Better to add a comment here indicating that we need to change the wait time back in the future.